tests/integration/Microsoft.Bot.Builder.Integration.Runtime.Tests/Resources/JsonResource.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Runtime.Tests.Resources
+{
+    public class JsonResource : MemoryResource
```

Comment:
why do we need all this additional test framework?

Reply:
I think to test declarative stuff without having it in json files but let me dig.

Reply:
We need the in-memory resources to ahve in-memory resource explorer instead of depending on having physical json declarative files.